### PR TITLE
code of conduct  with link

### PR DESCRIPTION
**Pull request by shigdel**
here i have added the link to code of conduct

### CODE REVIEW

File: 01.1-codeofconduct.Rmd [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Code of Conduct
+Version 1.0.2 (July 27, 2021)
+
+The Bioconductor community values an open approach to science that promotes the
+
+    * sharing of ideas, code, software and expertise
+    * collaboration
+    * diversity and inclusivity
+    * a kind and welcoming environment
+    * community contributions
```
Comment (antagomir) [inline 01.1-codeofconduct.Rmd:17]: Thanks for adding the bullet points here (based on earlier PR and its requests). 

But we have other similar lists later in this text. Well could you add the bullet points in the other relevant places as well?

File: 01.1-codeofconduct.Rmd [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+**Enforcement**
+
+Anyone asked to stop harassing or intimidating behavior are expected to comply immediately.
+
+If a person/s contravene the Code of Conduct the Code of Conduct committee retains the right to take any action that ensures a welcoming environment for all community members. This includes warning the alleged offender or temporary/permanent expulsion from the event and/or electronic platforms under Bioconductor’s control.
+
+The Code of Conduct committee may take action to redress anything designed to, or with the clear impact of, disrupting an event or electronic communication platform or making the environment hostile for any community member.
+
+We expect everyone in the Bioconductor community to comply with the Code of Conduct when participating in Bioconductor events and online communication platforms.
+Reporting
```
Comment (antagomir) [inline 01.1-codeofconduct.Rmd:58]: This seems to be a subtitle in the original text in 
https://bioconductor.github.io/bioc_coc_multilingual/

Can you check the original text, and make the appearance similar.. (titles are titles, bolding is bolding etc).

**Review by antagomir — CHANGES_REQUESTED** [review]
Kindly check the comments and modify accordingly. Pay attention that the rendered text is similar to the original version at https://bioconductor.github.io/bioc_coc_multilingual/

**Comment by TuomasBorman** [comment]
Also this is not yet added to the course material? What would be a good place? Before acknowledgements? https://microbiome.github.io/course_2022_oulu/index.html#acknowledgments

**Comment by antagomir** [comment]
> Also this is not yet added to the course material? What would be a good place? Before acknowledgements? https://microbiome.github.io/course_2022_oulu/index.html#acknowledgments

Just to check - in what sense it is not yet in the course material? (It is on the course website, or at least the link points there).

**Comment by TuomasBorman** [comment]
I rendered the book and it was not included in the book. I can check again

**Comment by antagomir** [comment]
@shigdel could you handle the feedback and commit the updates, so we can merge

File: 01.1-codeofconduct.Rmd (hunk 1 above)
Comment (antagomir) [reply 01.1-codeofconduct.Rmd:17]: Hi @shigdel - this seems like not resolved yet?

File: 01.1-codeofconduct.Rmd (hunk 2 above)
Comment (antagomir) [reply 01.1-codeofconduct.Rmd:58]: This also

**Review by antagomir — CHANGES_REQUESTED** [review]
Thanks - I do not see changes for the points that I asked in the review. Could you add and commit?

**Comment by antagomir** [comment]
Can this be closed now with #9

**Comment by TuomasBorman** [comment]
Yes

**Pull request closed without merging**